library/lib/views/image.cpp

```diff
@@ -280,6 +281,34 @@ void Image::setImageFromFile(std::string path)
     this->invalidate();
 }
 
+void Image::setImageFromMemory(unsigned char* data, int numData)
+{
+    NVGcontext* vg = Application::getNVGContext();
+
+    // Free the old texture if necessary
+    if (this->texture != 0)
+        nvgDeleteImage(vg, this->texture);
+
+    // Load the new texture
+    int flags     = this->getImageFlags();
+    this->texture = nvgCreateImageMem(vg, flags, data, numData);
+ 
+    if (this->texture == 0)
+    {    
+        std::stringstream addr;
+        addr << (void*)data;
+
+        fatal("Cannot load image from memory (" + std::to_string(numData) + " bytes at 0x" + addr.str() + ")");
```

Comment:
Why do you need a stringstream for that?

Reply:
To convert the address of the pointer to a std:: string (it needs to be a std::string so I can use the operator+). If there's an easier way though, let me know!

Reply:
I don't think that's worth just for an error message IMO